core/src/main/java/edu/wpi/grip/core/sources/CameraSource.java

```diff
@@ -154,8 +154,16 @@ public void start() throws IOException, IllegalStateException {
 
                     frameMat.copyTo(frameOutputSocket.getValue().get());
                     frameOutputSocket.setValue(frameOutputSocket.getValue().get());
-                    long thisMoment = System.currentTimeMillis();
-                    frameRateOutputSocket.setValue(1000 / (thisMoment - lastFrame));
+
+                    final long thisMoment = System.currentTimeMillis();
+                    final long elapsedTime = thisMoment - lastFrame;
+                    if (elapsedTime == 0) {
+                        // This can actually happen: https://github.com/WPIRoboticsProjects/GRIP/issues/199
+                        frameRateOutputSocket.setValue(Double.NaN);
```

Comment:
Brad offered a good suggestion. If for some reason the frame rate is zero then we should use the old value? Does this make sense?